erdpy/ledger/config.py

```diff
@@ -0,0 +1,41 @@
+import string
+
+
+def load_ledger_config_from_response(response: bytes):
+    config = ElrondLedgerAppConfiguration()
+
+    config.data_activated = False
+    if response[0] == 0x01:
+        config.data_activated = True
+
+    config.account_index = response[1]
+    config.address_index = response[2]
+
+    version = str(response[3]) + "." + str(response[4]) + "." + str(response[5])
+    config.version = version
+
+    return config
+
+
+def compare_versions(version1: string, version2: string) -> int:
```

Comment:
maybe you can add unit tests for this function to ensure is working as expected.

Reply:
added